changelog/seqera-enterprise/v24.1_major.mdx

```diff
@@ -0,0 +1,96 @@
+
+## Data Studios
+
+[Data Studios](https://docs.seqera.io/platform/24.2/data_studios) closes the loop from development to deployment and insights, allowing you to create, manage, and share notebook environments in Seqera with the click of a button. Data Studios also makes it seamless to work across teams with multi-user support, built-in authentication, and automatic snapshots as you work.
+
+- Host a combination of container images and compute environments for interactive analysis using your preferred tools, like Jupyter notebooks, RStudio, and Visual Studio Code IDEs.
+- Checkpoints provide a stable point-in-time snapshot.
+- Collaborate and share data in real time.
+
+Data Studios is available as a public preview. [Contact support](https://support.seqera.io) to request access to this feature. 
+
+## Data Explorer
+
+[Data Explorer](https://docs.seqera.io/platform/24.2/data/data-explorer) allows you to browse and interact with remote data repositories from organization workspaces in Seqera Platform. It supports AWS S3, Azure Blob Storage, and Google Cloud Storage repositories.
+
+Data Explorer is now Generally Available and supports multi-file and multi-folder download. This allows users to download entire datasets of pipeline results to share with their team.
+
+## Extended data roles
+
+User roles provide flexibility for admins to provide users with the permissions they need, without compromising security. The **Connect** user role has been added to the existing user roles with functionality related to Data Studios:
+
+- **Connect**: Open a connection to a running data studio session and interact with the contents.
+- **Maintain**: Create, update, start, stop, change configuration, and delete data studio sessions.
+- **View**: List, search, and view the status/configuration/details of data studios.
+- **Launch**: Open a connection to a running data studio session and interact with the contents.
+
+See [User roles](https://docs.seqera.io/platform/24.2/orgs-and-teams/roles) for more information.
+
+## Managed identities
+
+Managed identities offer significant advantages for High-Performance Computing (HPC) environments by enabling granular access control for individual workspace users. Unlike traditional SSH credentials that grant all workspace users access to HPC clusters with the same service account, managed identities ensure each user’s activity is logged under their own credentials.
+
+See [Managed identities](https://docs.seqera.io/platform/24.2/credentials/managed_identities) for more information. 
+
+## Other feature improvements
+
+- A redesigned [pipeline launch form](https://docs.seqera.io/platform/24.2/getting-started/quickstart-demo/launch-pipelines) provides an improved workflow and an interface to create and edit run parameters as raw JSON/YAML. Enable the new launch form per workspace via [environment variable](https://docs.seqera.io/platform/24.2/enterprise/configuration/overview#core-featuress).
```

Comment:
```suggestion
- A redesigned [pipeline launch form](https://docs.seqera.io/platform/24.2/getting-started/quickstart-demo/launch-pipelines) provides an improved workflow and an interface to create and edit run parameters as raw JSON/YAML. Enable the new launch form per workspace via [environment variable](https://docs.seqera.io/platform/24.2/enterprise/configuration/overview#core-features).
```

Cleaning up the failed anchor validation errors would avoid anymore instances of broken anchors.